src/test/resources/application.conf

```diff
@@ -16,48 +16,81 @@ mantis {
 
   network.peer.max-pending-peers = 1
 
-  network.discovery.bootstrap-nodes = [
-    "enode://375fc4e3712013120f392d909c57aead2461ba919130075e8fcd007e8326971a514741af0debff783ebdcb8c82ab72c8eee9a31adab629232e52d1e8e9b9b4b6@127.0.0.1:30340",
-    "enode://375fc4e3712013120f392d909c57aead2461ba919130075e8fcd007e8326971a514741af0debff783ebdcb8c82ab72c8eee9a31adab629232e52d1e8e9b9b4b7@10.0.0.0:30340"
-  ]
-
   network.rpc.apis = "eth,web3,net,personal,daedalus,debug"
 
-  blockchain {
-    eip155-block-number = "3000000"
+  blockchains {
```

Comment:
why not put this into `test.conf` file?

Reply:
Are you asking why it's not in `src/main/resources/chains/test.conf` or why it's not in `src/test/resources/test.conf`? 
The former - cause it's only for tests and thus I don't think it should present be in app resources
The latter - I don't think it's neccessary.